app/controllers/api/v2/users_controller.rb

```diff
@@ -96,7 +97,7 @@ def create
       api :PUT, "/users/:id/", N_("Update a user")
       description <<-DOC
         Adds role 'Default role' to the user if it is not already present.
-        Only another admin can change the admin account attribute.
+      Only another admin can change the admin account attribute.
```

Comment:
Please keep correct indenting.
```suggestion
        Only another admin can change the admin account attribute.
```